src/types/participant.ts

```diff
@@ -0,0 +1,43 @@
+import { Participant, ResponderStatus } from './activity';
+
+export function isActive(status: ResponderStatus) {
+  return [
+    ResponderStatus.Standby,
+    ResponderStatus.Remote,
+    ResponderStatus.SignedIn,
+    ResponderStatus.Available,
+    ResponderStatus.Assigned,
+    ResponderStatus.Demobilized
+  ].includes(status);
+}
+
+export function isCheckedIn(status: ResponderStatus) {
+  return [
+    ResponderStatus.Available,
+    ResponderStatus.Assigned
+  ].includes(status);
+}
+
+export const reduceActive = (count: number, participant: Participant) => {
```

Comment:
Many of these functions are also redefined above. Can we have just 1 implementation?

Reply:
Yeah, I'm planning to migrate the references to the other class, and deprecate the definitions in activities. There's a lot though and I my intent here was to get this functional for sharing. If we deploy this ill refactor the other refs.